### PR TITLE
fix: temporarily disable validateTypeDefs 

### PR DESCRIPTION
**Pull request by danstarns**
# Description

The typeDefs validation was causing too many issues and after @darrellwarde and I spent some time debugging we realised it would be detrimental to include it! This pr comments out the typedef validation :O. Removing the validation has no effect on the usability of the library and the only side effect would be users get a lesser defined error when they give us some invalid typeDefs. 

- [x] Documentation has been updated
- [x] TCK tests have been updated
- [x] Integration tests have been updated
- [x] Example applications have been updated
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

File: docs/asciidoc/auth/auth.adoc [hunk 1]
```diff
@@ -5,6 +5,8 @@ In this section you will learn more about how to secure your GraphQL API using N
 
 > Using auth requires various APOC functions and procedures, one being https://neo4j.com/labs/apoc/4.1/overview/apoc.util/apoc.util.validatePredicate/[apoc.util.validatePredicate] that is not currently in Neo4j Aura. Run the <<drivers-and-config-checkNeo4jCompat>> method to ensure your Graph has all the requirements.
 
+Using auth requires various APOC functions and producers, one being https://neo4j.com/labs/apoc/4.1/overview/apoc.util/apoc.util.validatePredicate/[apoc.util.validatePredicate] that is not currently in Neo4j Aura. Run the <<drivers-and-config-checkNeo4jCompat>> method to ensure your Graph has all the requirements.
```
Comment (darrellwarde) [inline docs/asciidoc/auth/auth.adoc:8]: Looks like this PR has added this note again.

**Review by darrellwarde — CHANGES_REQUESTED** [review]
Need to remove `auth.adoc` from this PR by the looks of things!

**Review by darrellwarde — APPROVED** [review]
Obviously not an ideal scenario, but looks good to me. Let's get it looked at again ASAP after release! ⏰

**Review by oskarhane — APPROVED** [review]
Unblocks the release 🍡

**Pull request merged**